new/PRS14224SilviaFrei.xml

```diff
@@ -45,7 +45,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="2">
-                    <persName xml:lang="de" xml:id="n1">Silvia Frei</persName>
+                    <persName><forename>Silvia</forename><surname>Frei</surname></persName>
```

Comment:
you need a space between forename and surname, or it will be printed SilviaFrei

Reply:
Done.